tests/models/water_network_simulation/test_rules_model_equivalence.py

```diff
@@ -0,0 +1,1136 @@
+                    "id": [103],  # PIPE3
+                    "operational.status": [False],
+                }
+            }
+        }
+        result_closed, _ = tester.update(3600, pipe_close_update)
+
+        # Verify PIPE3 flow is zero and flow is redistributed
+        if result_closed is not None:
+            pipe_data = result_closed.get("water_network", {}).get(
+                "water_pipe_entities", {}
+            )
+            if "drinking_water.flow" in pipe_data:
+                pipe3_flow = self._get_flow_by_id(pipe_data, 103)
+
+                if pipe3_flow is not None:
+                    # PIPE3 should have zero flow after closing
+                    assert abs(pipe3_flow) < 0.0001, (
+                        f"Closed pipe should have zero flow, got {pipe3_flow}"
+                    )
+
+                # PIPE1 should still have flow (serving J2 through PIPE2)
+                pipe1_flow = self._get_flow_by_id(pipe_data, 101)
+                if pipe1_flow is not None:
+                    assert pipe1_flow > 0, "PIPE1 should still have flow"
+
+                    # PIPE1 flow should be less than before (no longer serving J3)
+                    if initial_flows.get("pipe1") is not None:
+                        assert pipe1_flow < initial_flows["pipe1"], (
+                            f"PIPE1 flow should decrease after closing PIPE3"
+                        )
+
+
+class TestWNTRNetworkDirect:
+    """Direct WNTR tests to verify test network validity."""
+
+    def test_simple_network_converges(self):
+        """Verify that the simple test network converges in WNTR directly."""
+        wn = wntr.network.WaterNetworkModel()
+
+        # Add reservoir
+        wn.add_reservoir("R1", base_head=50.0, coordinates=(0.0, 0.0))
+
+        # Add junctions with small demands
+        wn.add_junction(
+            "J1", base_demand=0.001, elevation=10.0, coordinates=(100.0, 0.0)
+        )
+        wn.add_junction(
+            "J2", base_demand=0.001, elevation=10.0, coordinates=(200.0, 0.0)
+        )
+
+        # Add pipes
+        wn.add_pipe(
+            "P1",
+            start_node_name="R1",
+            end_node_name="J1",
+            length=100.0,
+            diameter=0.3,
+            roughness=100.0,
+        )
+        wn.add_pipe(
+            "P2",
+            start_node_name="J1",
+            end_node_name="J2",
+            length=100.0,
+            diameter=0.3,
+            roughness=100.0,
+        )
+
+        # Configure and run simulation
+        wn.options.time.duration = 3600
+        wn.options.time.hydraulic_timestep = 3600
+
+        sim = wntr.sim.WNTRSimulator(wn)
+        results = sim.run_sim()
+
+        # Verify results exist
+        assert len(results.node["pressure"]) > 0
+        assert results.node["pressure"].loc[0, "J1"] > 0
+        assert results.node["pressure"].loc[0, "J2"] > 0
+        assert results.link["flowrate"].loc[0, "P1"] > 0
```

Comment:
Does the existence of results say anything about convergence? Is there a way that WNTR reports on convergence (eg, residuals)?

Also, while it is good to have these results (and the results from the tests below), I don't think they have a place in our unit test suite, given that they don't test any of our code. We can use the results as validation of statements that we make in our documentation though!